samples/vmc/networks_nsxt/cgw_firewall_crud.py

```diff
@@ -39,7 +39,8 @@ def __init__(self):
         parser = argparse.ArgumentParser(
             formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
-        required_args = parser.add_argument_group('required arguments')
+        required_args = parser.add_argument_group(
```

Comment:
required_args is repeated in all the sample files. Can we declare it in library and use?

Reply:
fixed in the next commit

- new `sample_cli.py` file in the `helpers` directory
- all other samples will import `parser`, `required_args` and `optional_args` as needed